climada/engine/test/test_forecast.py

```diff
@@ -162,6 +155,21 @@ def test_Forecast_plot(self):
                     for f in source:
                         if f['properties']['adm0_a3'] == 'CHE':
                             sink.write(f)
+        #test plotting functions
+        forecast.plot_imp_map(run_datetime=dt.datetime(2017,12,31),
+                              polygon_file=str(cantons_file),
+                              save_fig=True,close_fig=True)
+        map_file_name = (forecast.summary_str(dt.datetime(2017,12,31)) +
+                         '_impact_map' +
+                         '.jpeg')
+        map_file_name_full = Path(FORECAST_PLOT_DIR) / map_file_name
+        map_file_name_full.absolute().unlink(missing_ok=True)
```

Comment:
Should it not fail if the file was not created?
```suggestion
        map_file_name_full.absolute().unlink(missing_ok=False)
```